mappings/net/minecraft/client/sound/SoundEngine.mapping

```diff
@@ -22,6 +24,9 @@ CLASS net/minecraft/class_4225 net/minecraft/client/sound/SoundEngine
 	METHOD method_38501 openDeviceOrFallback (Ljava/lang/String;)J
 		ARG 0 deviceSpecifier
 	METHOD method_38502 getSoundDevices ()Ljava/util/List;
+	METHOD method_38722 getCurrentDeviceName ()Ljava/lang/String;
```

Comment:
```suggestion
	METHOD method_38722 getDeviceName ()Ljava/lang/String;
```
No point including `current` if there is no contrast with another "old" device, etc.

Reply:
There is, kinda; this will always return up-to-date info from the OS, and is used inside a reload code (to check if system default has changed). The name is not good, I guess, but "getDeviceName" sounds like a simple field getter when it isn't.